info.json

```diff
@@ -1,17 +1,19 @@
 {
-  "name": "colorblind_ultimate",
-  "version": "2.1.0",
-  "title": "Color Blind Ultimate",
-  "author": "rycieos",
-  "factorio_version": "2.0",
-  "dependencies": [
-      "? base >= 2.0.7",
-      "? elevated-rails >= 2.0.7",
-      "? quality >= 2.0.7",
-      "? space-age >= 2.0.7",
-      "? Ultracube >= 0.2.0"
-  ],
-  "description": "Ultimate options for color blindness accessibility. Includes custom icons, sprites, icon and sprite overlays, text overlays, customizable wire colors, customizable game and map overlay colors, and customizable technology background colors. Everything is individually customizable. Can be removed without affecting save games.",
-  "homepage": "https://github.com/Rycieos/factorio-colorblind-ultimate",
-  "contact": "https://github.com/Rycieos/factorio-colorblind-ultimate/issues/new/choose"
+    "name": "colorblind_ultimate",
+    "title": "♿Color Blind Ultimate ♿",
+    "description": "♿ Ultimate options for color blindness accessibility. \n  Includes custom icons, sprites, icon and sprite overlays, text overlays, customizable wire colors, customizable game and map overlay colors, and customizable technology background colors. Everything is individually customizable. \n Can be removed without affecting save games.",
+
+    "author": "rycieos",
+    "homepage": "https://github.com/Rycieos/factorio-colorblind-ultimate",
+    "contact": "https://github.com/Rycieos/factorio-colorblind-ultimate/issues/new/choose"
+
+    "version": "2.1.1",
```

Comment:
In general, I would prefer PRs to not bump version info. Not only might I want to add other changes to the release, but if this isn't merged before another release happens, then this PR would have a merge conflict.

Reply:
It depends on the github, sometimes the devs like us to do a complete and careful job, others don't care, personally I avoid adding work to other people,

Reply:
Understood. And I can't be upset since I haven't added development guides to this project, so you have no way of knowing ahead of time. But I am telling you now. I have another change ready that I want to go in the same version, and it will not be a patch version bump.